zebra-chain/src/transaction/serialize.rs

```diff
@@ -182,6 +180,23 @@ impl ZcashSerialize for Transaction {
                     None => {}
                 }
             }
+            Transaction::V5 {
+                inputs,
+                outputs,
+                lock_time,
+                expiry_height,
+                rest,
+            } => {
+                // Write version 5 and set the fOverwintered bit.
+                writer.write_u32::<LittleEndian>(5 | (1 << 31))?;
+                writer.write_u32::<LittleEndian>(TX_V5_VERSION_GROUP_ID)?;
+                inputs.zcash_serialize(&mut writer)?;
+                outputs.zcash_serialize(&mut writer)?;
+                lock_time.zcash_serialize(&mut writer)?;
+                writer.write_u32::<LittleEndian>(expiry_height.0)?;
```

Comment:
v5 transaction fields are in a different order, see https://github.com/daira/zips/raw/orchard-circuit/protocol/orchard.pdf

![BEE1D0AE-FF15-45DD-96E1-6122CC4F3C56](https://user-images.githubusercontent.com/8951843/109713067-7deb4100-7bec-11eb-9f24-8f16117393bc.jpeg)

Reply:
Do you think we should change the order also in `zebra-chain` -> `Transaction::V5` too ?

Reply:
Yes, I think it's helpful to have the fields in serialized order, to avoid future mistakes or confusion.